src/shared/ethtool-util.c

```diff
@@ -438,8 +438,22 @@ int ethtool_set_nic_buffer_size(int *ethtool_fd, const char *ifname, netdev_ring
                 }
         }
 
+        if (ring->rx_mini_pending_set) {
+                if (ecmd.rx_mini_pending != ring->rx_mini_pending) {
```

Comment:
why nest two if checks? why not merge the condition in one? we generally try to keep indentation levels low

Reply:
Because that's how the previous code did it and that passed review, which usually means that's the way to go.

Merged the two ifs, with another commit before that handles updating the old code.